.github/workflows/ruby.yml

```diff
@@ -92,7 +92,6 @@ jobs:
         ports:
           - 27017:27017
     env:
-      JRUBY_OPTS: "--1.9"
```

Comment:
`--1.9` is ignored.
https://github.com/mongomapper/mongomapper/runs/6824975680?check_suite_focus=true#step:6:9